app-admin/restart-services/restart-services-0.17.0-r1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Tool to manage OpenRC and systemd services that need to be restarted"
+HOMEPAGE="https://dev.gentoo.org/~mschiff/restart-services/"
+SRC_URI="https://dev.gentoo.org/~mschiff/src/${PN}/${P}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND="
+	app-admin/lib_users
+	app-portage/portage-utils
+	|| ( sys-apps/openrc sys-apps/systemd )
+"
+
+src_install() {
+	dosbin restart-services
+	doman restart-services.1
+	keepdir /etc/restart-services.d
+	insinto /etc
+	doins restart-services.conf
+	dodoc README CHANGES
+
+	sed -i 's/^#include/include/' "${D}"/etc/restart-services.conf
+	cat>"${D}"/etc/restart-services.d/00-local.conf<<-EOF
+	# You may put your local changes here or in any other *.conf file
+	# in this directory so you can leave /etc/restart-services.conf as is.
+	# Example:
+	# *extend* SV_ALWAYS to match 'myservice'
+	# SV_ALWAYS+=( myservice )
+	EOF
+}
+
+pkg_postinst() {
+	local MAJOR MINOR
+	# migrate config data for versions < 0.13.2
+	if [[ $REPLACING_VERSIONS ]]; then
+		MAJOR=${REPLACING_VERSIONS%%.*}
+		MINOR=${REPLACING_VERSIONS%.*}
+		MINOR=${MINOR#*.}
```

Comment:
I suspect you can use here the `ver_cut`, `ver_test` for those checks here

Reply:
Actually, is it necessary?, this is the only version available.

Reply:
Yeah, should be okay to drop it by now if it was removed ages ago.